index.js

```diff
@@ -1,7 +1,7 @@
 
 require("dotenv").config();
 
-// docker run -rm --name my-mongo-db -dp 27017:27017 -v mongo-data:data/db mongo"docker 
+// docker run -rm --name my-mongo-db -dp 27017:27017 -v mongo-data:/data/db mongo"docker 
```

Comment:
What's with the double quote at the end of the line?

Reply:
No idea. I have deleted it.